server/app/views/applicant/ApplicantUpsellCreateAccountView.java

```diff
@@ -94,25 +63,104 @@ public Content render(
                     .withClasses(ReferenceClasses.BT_APPLICATION_ID, "mb-4"))
             .with(div(customConfirmationMessage.getOrDefault(locale)).withClasses("mb-4"));
 
+    Modal loginPromptModal = createLoginPromptModal(messages, redirectTo);
+
     // Don't show "create an account" upsell box to TIs, or anyone with an email address already.
-    if (Strings.isNullOrEmpty(account.getEmailAddress()) && account.getMemberOfGroup().isEmpty()) {
-      content.with(createAccountSection);
+    boolean hasEmailAddress = !Strings.isNullOrEmpty(account.getEmailAddress());
```

Comment:
Is the email address check here to see if they already have an account? Is so, you probably want to be checking if the account has an authority_id instead.